app/tests/src/shelley.js

```diff
@@ -1,106 +0,0 @@
-/* eslint-disable max-len */
-import assert from 'assert'
-
-import ShelleyJsCryptoProvider from '../../frontend/wallet/shelley/shelley-js-crypto-provider'
-
-import {ShelleyBaseAddressProvider} from '../../frontend/wallet/shelley/shelley-address-provider'
-import {buildTransaction} from '../../frontend/wallet/shelley/helpers/chainlib-wrapper'
-import mnemonicToWalletSecretDef from '../../frontend/wallet/helpers/mnemonicToWalletSecretDef'
-import loadWasmModule from './loadWasmModule'
-import {NetworkId} from '../../frontend/wallet/types'
-
-const getCryptoProvider = async (mnemonic, networkId) => {
-  const walletSecretDef = await mnemonicToWalletSecretDef(mnemonic)
-  const network = {
-    networkId,
-  }
-  return await ShelleyJsCryptoProvider({
-    walletSecretDef,
-    network,
-    config: {shouldExportPubKeyBulk: true},
-  })
-}
-
-// inform mocha global leak detector
-window.wasm = null
-before(loadWasmModule)
-
-describe('shelley address derivation', () => {
-  const mnemonic15Words =
-    'abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon address'
-  it('should derive base address from 15-words mnemonic', async () => {
```

Comment:
this test seems to still be testing live code (derivation of a single base address), so I'd keep it

Reply:
It does, but it tests something that is already tested elsewhere so they are redundant.

Reply:
I see the full manager being tested, not the individual address providers, but maybe I'm just missing something. If not, this IMHO still serves to test the address provider in isolation which may come in handy for refactors related to the address provider alone, decoupling address-provider-level errors from the address-manager ones

Reply:
Fair point, fixed.